test/functional/files.js

```diff
@@ -0,0 +1,48 @@
+const variants = [
+  'default',
+  'translations',
+  'links',
+  'images',
+  'videos-iframe',
+  'videos-tag',
+];
+
+describe('files', () => {
+  before(() => {
+    // Set viewport size
+    browser.setViewportSize({
+      width: 1400,
+      height: 600,
+    });
+
+    browser.pause(1000);
+  });
+
+  variants.forEach(variant => {
+    describe(`--${variant}`, () => {
+      before(() => {
+        // Go to url
+        browser.url(`ecl-files--${variant}.html`);
+        // Make sure the browser has finished painting
+        browser.pause(1000);
+        // Inject axe-core (for accessibility tests)
+        browser.injectAxeCore();
+      });
+
+      // Normal state
+      context('with plain state', () => {
+        it('should match the reference screenshot', () => {
+          const screenshots = browser.checkDocument({
+            name: `files/${variant}`,
+          });
+          expect(screenshots).to.matchReference();
+        });
+
+        it('should be accessible', () => {
+          const a11yReport = browser.runAxeCore('ecl-files').value;
```

Comment:
Beware! It should be `'ecl-file'` and not `'ecl-files'`. That's the reason why the a11y tests are not run